testsuites/syncgateway/functional/topology_specific_tests/multiple_servers/test_multiple_servers.py

```diff
@@ -101,7 +101,7 @@ def test_rebalance_sanity(params_from_base_test_setup):
 @pytest.mark.session
 @pytest.mark.channel
 @pytest.mark.bulkops
-@pytest.mark.skip(reason="Failing due to - https://github.com/couchbase/sync_gateway/issues/2197")
+# @pytest.mark.skip(reason="Failing due to - https://github.com/couchbase/sync_gateway/issues/2197")
```

Comment:
If the test is working, can the comment be removed?

Reply:
this test is failing even with multiple url support, Assign to Adam. Adam is looking into it

Reply:
will enable once Adam/ Traun fix it

Reply:
If the test is failing, then the @pytest.mark.skip should not be commented right?